CGAL_ipelets/include/CGAL/CGAL_Ipelet_base_v7.h

```diff
@@ -66,20 +66,20 @@ namespace CGAL{
     ipe::TSelect get_selection_type() const { return get_IpePage()->primarySelection()==-1 ? ipe::EPrimarySelected : ipe::ESecondarySelected;}  
     //ipe6 compatibility
     void transform_selected_objects_(const IpeMatrix& tfm) const {
-      for (int i=0;i<get_IpePage()->count();++i)
+      for (int i=0;i<static_cast<int>(get_IpePage()->count());++i)
         if (get_IpePage()->select(i)!=ipe::ENotSelected)
           get_IpePage()->transform(i,tfm);      
     }
  
     void delete_selected_objects_() const {
-      for (unsigned i=get_IpePage()->count();i>0;--i)
+      for (unsigned i=static_cast<int>(get_IpePage()->count());i>0;--i)
```

Comment:
Really, even here?

Reply:
That cast is ugly, but:
  - the return type of `count()` changes from one version of Ipe to another,
  - that return type is not a nested typedef (like `size_type` in STL containers),
  - and `INT_MAX` is enough for the page count of any document.

Reply:
But why not be consistent and either cast to unsigned (the type of i) or make i an int (the type you are casting to)? Or wait a month and use `auto`...

Reply:
You are right!...

@maxGimeno Marc is right: in that line the loop variable `i` has type `unsigned int`, and that is not consistent with the cast. And anyway that loop is different from the other ones. Be careful.

Reply:
Oh yes I see. Sorry, I just used "replace all" without reading carefully... My bad.